watertap/core/membrane_channel_base.py

```diff
@@ -449,8 +485,17 @@ def eq_cp_modulus(b, t, x, j):
                 doc="Membrane channel mass transfer coefficient",
             )
 
+            if sherwood_number_eq not in (
+                SherwoodNumberEq.length_dependent,
+                SherwoodNumberEq.length_independent,
+            ):
+                raise ConfigurationError(
+                    f"Unrecognized sherwood_number_eq {sherwood_number_eq}"
+                )
+
             if mass_transfer_coefficient == MassTransferCoefficient.calculated:
                 self._add_calculated_mass_transfer_coefficient()
+                self._add_sherwood_number()
```

Comment:
@bknueven Do I need to pass in the sherwood number like this or can I just set it like `sherwood_number_eq=SherwoodNumberEq.length_independent` and get rid of the _add_sherwood_number function?

Either way, is it still necessary to keep the `sherwood_number_eq=SherwoodNumberEq.length_independent` in the add_calculated_mass_transfer_coefficient?

Reply:
You don't need to have an `_add_sherwood_number` method. You just need to pass the value of `sherwood_number_eq` into `_add_calculated_mass_transfer_coefficient` and add the appropriate sherwood number equation there (like you were doing before).